pkg/ccl/importccl/import_processor_test.go

```diff
@@ -102,7 +102,7 @@ func TestConverterFlushesBatches(t *testing.T) {
 				group := ctxgroup.WithContext(ctx)
 				group.Go(func() error {
 					defer close(kvCh)
-					return conv.readFiles(ctx, testCase.inputs, converterSpec.Format, externalStorageFactory)
+					return conv.readFiles(ctx, testCase.inputs, nil, converterSpec.Format, externalStorageFactory)
```

Comment:
The style guide, for better or worse, discourages naked literals like `nil` or `false` where their meaning isn't plainly obvious i.e. as the 3 arg to some method. It suggests adding a trailing inline comment e.g. 
`doFoo(bar, baz, nil /* fooOptions */, false /* overwrite */)`. Personally, I find those inline comments noisy and dislike them, so I typically prefer to make a named constant and pass that instead, e.g. at the top of my package (or immediately above the call if it is the only one) I add `const overwrite, noOverwrite = true, false` and `const defaultOpts = nil` is then the callsites look like `doFoo(bar, baz, defaultOpts,  noOverwrite)`. IMO this pattern works particularly well when you have many calls that pass nil or false or whatever.

If I think many callers will need to pass nil/false/whatever for an exported, public API I often even go ahead and add a named exported constant right along side the definition.

However we often don't impose this rule on test code, so this is also OK as-is -- just wanted to take the chance to point it out.

Reply:
Thanks -- I totally understand this style guide.  For what it's worth, I didn't bother with anything other than nil because the subsequent PRs do modify call site to pass in appropriate resume map.